py_data_rules/column.py

```diff
@@ -0,0 +1,24 @@
+from .data_type import DataType, XSDString
```

Comment:
I would argue that the full package path is better than relative imports.
But if there are no import errors, I guess its just a design choice